test/main/test.js

```diff
@@ -234,19 +234,23 @@ QUnit.module( "test", function() {
 
 	( function() {
 		var previousTestAssert;
+		var firstTestName = "assertions after test finishes throws an error - part 1";
 
 		QUnit.module( "bad assertion context" );
 
-		QUnit.test( "assertions after test finishes throws an error - part 1", function( assert ) {
+		QUnit.test( firstTestName, function( assert ) {
 			assert.expect( 0 );
 			previousTestAssert = assert;
 		} );
 
 		QUnit.test( "assertions after test finishes throws an error - part 2", function( assert ) {
 			assert.expect( 1 );
 			assert.throws( function() {
-				previousTestAssert.ok( true );
-			}, /Assertion occurred after test had finished/ );
+				previousTestAssert.ok( true, "message here" );
+			}, "Assertion occurred after test had finished.\n" +
```

Comment:
This string is used as message, not as expected value matcher. I apologize, this is ambiguous indeed due to both second and third parameter being optional. Support for literal string matching against `error.message` was removed for that reason. Either use a regex for a partial match, or use an Error object for a literal match.

E.g.

```js
assert.throws( function () {
  /* … */
}, new Error( "foo" ), "optional message" );
```

<https://api.qunitjs.com/assert/throws/>

Reply:
Ahh entirely my bad, I think I have that fixed now.